mantidproject/mantid
# Fit to raw checkbox now works as intended

**Description of work.**
This pull request fixes the isssue with the fit to raw checkbox in muon guis unchecking when no rebinned  workspaces are present.

**To test:**
1. Open muon analysis gui
2. load current run
3.  Try unchecking fit to raw checkbox this should give a warning 
4. Trying to uncheck the fit to raw checkbox again should give a warning no matter how many time you try when no rebined data is present.



Fixes #32187. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
